doc/userdoc/guides/built-in_timers.rst

```diff
@@ -0,0 +1,86 @@
+Built-in timers
+===============
+
+Basic timers
+------------
+
+Basic built-in timers keep track of the time NEST spent for network construction and actual simulation (propagation of the network state). These timers are active in all simulations with NEST, and the measured times can be queried using ``GetKernelStatus``. For example:
+
+::
+
+    nest.GetKernelStatus('time_simulate')
+
+The following basic time measurements are available:
+
++-----------------------------+----------------------------------+
+|Name                         |Explanation                       |
++=============================+==================================+
+|``time_construction_create`` |Cumulative time NEST spent        |
```

Comment:
Will it be clear to users what is meant by "cumulative"? E.g. a note like

"Cumulative measurements are collected from the start of NEST or the most recent call to ``ResetKernel``."